packages/i18n/src/locales/en/translations.ts

```
@@ -1703,6 +1819,11 @@ export default {
         error: "Project could not be updated. Please try again.",
       },
     },
+    features: {
+      updating: "Updating project feature...",
+      updated: "Project feature updated successfully.",
+      error: "Something went wrong while updating project feature. Please try again.",
+    },
```

Biome (2.3.14)

[error] This property is later overwritten by an object member with the same name.

Overwritten with this property.

If an object property with the same name is defined multiple times (except when combining a getter with a setter), only the last definition makes it into the object and previous definitions are ignored.
Unsafe fix: Remove this property.


(lint/suspicious/noDuplicateObjectKeys)